moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -88,13 +89,15 @@ class PlanningScene : private boost::noncopyable, public std::enable_shared_from
 {
 public:
   /** \brief construct using an existing RobotModel */
-  PlanningScene(const robot_model::RobotModelConstPtr& robot_model,
-                collision_detection::WorldPtr world = collision_detection::WorldPtr(new collision_detection::World()));
+  PlanningScene(
```

Comment:
This looks like clang-format wasn't applied here (maybe wrong version).
Also the smart pointer world should be passed by value and not by const reference.